src/Commands/BannedKeywordsBag.ts

```diff
@@ -0,0 +1,203 @@
+/**
+ * A container for banned keywords for variable names.
+ */
+export class BannedKeywordsBag {
+    /**
+     * Banned keywords, keyed in alphabetical order.
+     */
+    private bannedKeyWords: { [i: string]: Boolean };
+
+    /**
+     * Initalizes a new instance of the BannedKeywordsBag class.
+     */
+    constructor() {
+        this.bannedKeyWords = {
+            "BEGIN": false,
```

Comment:
Lets do `true` as the values. That's semantically closer to what we want: an indication that they are indeed banned.